apprentice_teacher.rb

```diff
@@ -1,21 +1,17 @@
-class ApprenticeTeacher
+require_relative 'instructors'
+require_relative 'high_five'
+
+class ApprenticeTeacher < Instructor
+
+PERFORMANCE_RATING = 80
+
+include HighFive
   attr_reader :age, :salary, :phase, :target_raise
   attr_accessor :name
 
   def initialize(options={})
-    @age = options.fetch(:age, 0)
-    @name = options.fetch(:name, "")
+    super
     @target_raise = 800
```

Comment:
could this also be a constant? I notice this attribute appears in a few classes. How could you abstract this out into the parent class?